### PR TITLE
Add order api

### PR DESCRIPTION
**Pull request by mhdtahawi**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
+plugins {
+    id 'java'
+    id "io.freefair.lombok" version "5.0.0-rc6"
+}
+
+group 'krakenApi'
+version '1.0-SNAPSHOT'
+
+
+sourceCompatibility = 1.8
```
Comment (osenseijedi) [inline build.gradle]: should be 1.11 from now on

File: build.gradle [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile 'com.fasterxml.jackson.core:jackson-databind:2.8.9'
```
Comment (osenseijedi) [inline build.gradle]: All deps should be "implementation".
Also should we upgrade all libs versions here too ?

File: build.gradle (hunk 1 above)
Comment (mhdtahawi) [reply build.gradle]: 1.11 or just 11 ?

File: src/main/java/com/ptrader/connector/kraken/input/AddStandardOrderInput.java [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+        }
+
+
+        @Override
+        public String toString() {
+            return (isOffset ? "+" : "") + time.toString();
+        }
+    }
+
+    private AddStandardOrderInput(String pair, TransactionType type, double volume, boolean validate) {
```
Comment (osenseijedi) [inline src/main/java/com/ptrader/connector/kraken/input/AddStandardOrderInput.java]: double: we should use BigDecimal instead.

File: src/main/java/com/ptrader/connector/kraken/input/AddStandardOrderInput.java [hunk 4]
```diff
@@ -0,0 +1,289 @@
+package com.ptrader.connector.kraken.input;
+
+import com.ptrader.connector.kraken.input.exceptions.InvalidStateException;
+import lombok.Getter;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+@Getter
+public class AddStandardOrderInput implements ApiInput {
```
Comment (osenseijedi) [inline src/main/java/com/ptrader/connector/kraken/input/AddStandardOrderInput.java:14]: I would actually separate the object itself and its factory

File: build.gradle (hunk 1 above)
Comment (osenseijedi) [reply build.gradle]: 1.11

**Review by osenseijedi — APPROVED** [review]

**Pull request merged**